main.rb

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env ruby
+
+# Create methods that do the following:
+# List all books.
+# List all people.
+# Create a person (teacher or student, not a plain Person).
+# Create a book.
+# Create a rental.
+# List all rentals for a given person id.
+
+require './classroom'
+require './person'
+require './teacher'
+require './student'
+
```

Comment:
- [ ] Your app does not work the way it should. It will be nice to go through your [project requirement](https://www.youtube.com/watch?v=vkkgrhD6aXQ) again to get the full knowledge of the project requirement. Currently, a user can not create a book, a person, or a rental. 

- [ ] Kindly create a class `App` with a constructor(initialize) method where you can have books, people, and rentals instance variables so that you can use them to store the books, people, and rentals created by a user. Then move all the methods you have in this file except the `main` into the class so that all the methods can have access to the instance variables.

See suggestion:
```ruby
def initialize
    @books = []
    @people = []
    @rentals = []
  end
```